Minio.Examples/Minio.Examples.csproj

```diff
@@ -1,15 +1,15 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <OutputType>Exe</OutputType>
     <IsPackable>False</IsPackable>
   </PropertyGroup>
-
   <ItemGroup>
-    <PackageReference Include="Polly" Version="7.2.0" />
+    <PackageReference Include="Polly" Version="7.2.0"/>
+    <PackageReference Include="System.Net.Http" Version="4.3.0"/>
+    <PackageReference Include="System.Net.Primitives" Version="4.3.0"/>
```

Comment:
this doesn't seem to be used and can be removed, so this file need not be modified at all